notebooks/official/training/pytorch_gcs_data_training.ipynb

```diff
@@ -0,0 +1,1173 @@
+{
```

Comment:
<p>BUCKET_PREFIX doesn't seem to be used anywhere else.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1705/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Changed. I had intended to use it later, for the output of the model.</p>